src/components/tag/Tag.vue

```diff
@@ -33,7 +34,8 @@
         v-else
         class="tag"
         :class="[type, size, { 'is-rounded': rounded }]">
-        <span :class="{ 'has-ellipsis': ellipsis }">
+        <b-icon custom-class="" v-if="icon" :icon="icon" :size="size" :type="iconType" :pack="iconPack" />
```

Comment:
Please remove `custom-class`

Reply:
Thanks for letting me know, going to do the changes in the upcoming days.